t/test-lib-functions.sh

```diff
@@ -754,14 +767,32 @@ projfs_start () {
 	trap projfs_stop EXIT
 }
 
+# Wait for the "$1" to appear in the list of mounts.
+projfs_manual_wait_mount() {
+	tries_left=30
+	sleep_interval=0.1
+	while ! (mount | grep $1 >/dev/null)
+	do
+		tries_left=$((tries_left - 1))
+		if test $tries_left -eq 0
+		then
+			return 1
+		fi
+		sleep $sleep_interval
+	done
+}
```

Comment:
@chrisd8088 I'd particularly like your review of this "alternate" wait mount strategy. It should be a reasonably reliable alternative to the `stat`-based version which triggers projection.

Reply:
In light of 1488874, not needed!